src/components/charts/PMChart.svelte

```diff
@@ -12,6 +12,32 @@
   let pm2_5Data = [];
   let pm10Data = [];
 
+  $: getPMData(readings);
+
+  function getPMData(readings) {
+    pm1Data = readings.map((reading) => {
+      const d = parseISO(reading.captured, DATE_TIME_FORMAT_KEY);
+      return {
+        x: format(d, DATE_TIME_FORMAT_KEY),
+        y: parseFloat(reading.pm01_0).toFixed(2),
+      };
+    });
+    pm2_5Data = readings.map((reading) => {
+      const d = parseISO(reading.captured, DATE_TIME_FORMAT_KEY);
+      return {
+        x: format(d, DATE_TIME_FORMAT_KEY),
+        y: parseFloat(reading.pm02_5).toFixed(2),
+      };
+    });
+    pm10Data = readings.map((reading) => {
+      const d = parseISO(reading.captured, DATE_TIME_FORMAT_KEY);
```

Comment:
Just curious, why do we need to both parse and then re-generate the date string? Consistency perhaps?

Reply:
Yes, mostly consistency and legacy code elsewhere in the project that established this pattern.

The original `reading.captured` string comes in as something like `2022-08-22T06:27:38Z`, then the `date-fns` library's `parseISO()` turns it into `Mon Aug 22 2022 02:27:38 GMT-0400 (Eastern Daylight Time)`, and finally we can take that consistent date object and `format()` the pieces necessary for the particular component (the tooltip display in this case) which displays dates as `MM-dd HH:mm`

Reply:
Thanks for clarifying. It would be good to have that in the comments for our future selves.

Reply:
Will do.